UnityProject/Assets/Prefabs/Items/Clothing/Earwear/RadioHeadset.prefab

```diff
@@ -155,7 +155,10 @@ PrefabInstance:
       value: 
       objectReference: {fileID: 21300000, guid: 70f387a63a6b45d47b7417b73f4a2352,
         type: 3}
-    m_RemovedComponents: []
+    m_RemovedComponents:
+    - {fileID: 166801742205184598, guid: 23a0b694284152342b2ec0817fade0d6, type: 3}
```

Comment:
what are the removed components?

Reply:
for some reason headsets are children of stationbound radios
so they shouldn't get scripts from that

Reply:
The best solution would be to change the SBR to a _BaseRadio prefab and then make a new prefab variant for SBR. That way we wouldn't run into a situation where we have to remove components that we add to station balanced radios that we don't want on headsets.